Documentation/Specification/specification.md

```diff
@@ -126,19 +126,87 @@ For example, 8FVC9G8F+6W has the center 47.365562,8.524813. The following table
 
-Open Location Code library implementations must provide:
-* a method to convert a latitude and longitude into a 10 digit Open Location Code
-* a method to decode a 10 digit Open Location Code into, at a minimum, the latitude and longitude of the south-west corner and the height and width
-* a method to determine if a string is a valid sequence of Open Location Code characters
-* a method to determine if a string is a valid full Open Location Code
-* decode and validation methods must not be case-sensitive
+Being able to say _WF8Q+WF, Praia_ is significantly easier than remembering and using _796RWF8Q+WF_.
+With that in mind, how do you choose the locality to use as a reference?
 
-Open Location Code library implementations can provide:
-* a method to convert a latitude and longitude into any valid length Open Location Code
-* a method to decode any valid length Open Location Code, providing additional information such as center coordinates
-* a method to to convert a valid Open Location Code into a short code, given a reference location
-* a method to recover a full Open Location Code from a short code and a reference location.
-* a method to determine if a string is a valid short Open Location Code
+Ideally, you need to use both the bounding box of the locality, as well as its center point.
 
+Given a global code, _796RWF8Q+WF_, you can eliminate the first **four** digits of the code if:
+ * The center point of the feature is within **0.4** degrees latitude and **0.4** degrees longitude
+ * The bounding box of the feature is less than **0.8** degrees high and wide.
+
+(These values are chosen because a four digit Open Location Code is 1x1 degrees.)
+
+If there is no suitable locality close enough or small enough, you can eliminate the first **two** digits of the code if:
+ * The center point of the feature is within **8** degrees latitude and **8** degrees longitude
+ * The bounding box of the feature is less than **16** degrees high and wide.
+
+(These values are chosen because a two digit Open Location Code is 20x20 degrees.)
+
+The values above are slightly smaller than the maximums to allow for different geocoder backends placing localities in slightly different positions.
+Although they could be increased there will be a risk that a shortened code will recover to a different location than the original, and people misdirected.
+
+Note: Usually your feature will be a town or city, but you could also use geographical features such as lakes or mountains, if they are the best local reference.
+If a settlement (such as neighbourhood, town or city) is to be used, you should choose the most prominent feature that meets the requirements, to avoid using obscure features that may not be widely known.
+(Basically, prefer city or town over neighbourhood.)
+
+## API Requirements
+
+The following public methods should be provided by any Open Location Code implementation, subject to minor changes caused by language conventions.
```

Comment:
This section is labeled "requirements" but then contradicts that using "should" below.

Please consider RFC 2119 language including "MUST".

Reply:
PR is already merged.